service/grpc/codelingo/codelingo.pb.go

```diff
@@ -108,20 +80,6 @@ func (m *DescribeFactReply) String() string            { return proto.CompactTex
 func (*DescribeFactReply) ProtoMessage()               {}
 func (*DescribeFactReply) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{1} }
 
```

Comment:
exported method DescribeFactReply.GetProperties should have comment or be unexported